src/GNNGraphs/generate.jl

```diff
@@ -362,3 +362,96 @@ function rand_temporal_radius_graph(number_nodes::Int,
     end
     return TemporalSnapshotsGNNGraph(tg)
 end
+
+
+function _hyperbolic_distance(nodeA::Array{Float64, 1},nodeB::Array{Float64, 1}; ζ::Real)
+    if nodeA!=nodeB
+        d=acosh(cosh(ζ*nodeA[1])*cosh(ζ*nodeB[1])-(sinh(ζ*nodeA[1])*sinh(ζ*nodeB[1])*cos(pi-abs(pi-abs(nodeA[2]-nodeB[2])))))/ζ
```

Comment:
```suggestion
    if nodeA != nodeB
        d=acosh(cosh(ζ*nodeA[1])*cosh(ζ*nodeB[1])-(sinh(ζ*nodeA[1])*sinh(ζ*nodeB[1])*cos(pi-abs(pi-abs(nodeA[2]-nodeB[2])))))/ζ
```
whitespaces around operators missing everywhere and the long ling should be broken into multiple ones for readability